config/notifiers.go

```diff
@@ -503,11 +503,6 @@ func (c *WebhookConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if c.URL != nil && c.URLFile != "" {
 		return fmt.Errorf("at most one of url & url_file must be configured")
 	}
-	if c.URL != nil {
```

Comment:
We think this is duplicate code as `SecretURL` is just a `URL`, whose `UnmarshalYaml` function also calls `parseURL` which contains the same check.